crates/wasi-http/src/http_impl.rs

```diff
@@ -43,6 +43,18 @@ fn port_for_scheme(scheme: &Option<Scheme>) -> &str {
     }
 }
 
+fn is_allowed(allow_list: &Vec<String>, value: String) -> bool {
+    for allowed in allow_list.iter() {
+        if allowed == "*" {
```

Comment:
Do you want to follow the convention in [wasi-experimental-http](https://github.com/deislabs/wasi-experimental-http/blob/8291baece45cc51e18e69d7d5ad39ca20744e9f9/crates/wasi-experimental-http-wasmtime/src/lib.rs#L16) that uses "insecure:allow-all" to match everything?